catalyst/utils/initialization.py

```diff
@@ -75,4 +75,17 @@ def outer_init(layer: nn.Module) -> None:
             nn.init.uniform_(layer.bias.data, -v, v)
 
 
-__all__ = ["get_optimal_inner_init", "outer_init"]
+def weight_reset(module: nn.Module):
```

Comment:
`reset_weights`? btw, do we really need this method? could we just use `module.reset_parameters()`

Reply:
in this case we need to create loop like
```python
for _name, module in model.named_parameters():
    try:
        module.reset_parameters()
    except AttributeError as ex:
        pass
```
Instead of `model.apply(weight_reset)`
I think the second variant is more preferable.